playwright.config.ts

```diff
@@ -13,7 +13,7 @@ config();
 export default defineConfig({
   testDir: './tests/e2e',
   /* Maximum time one test can run for. */
-  timeout: 10 * 1000,
+  timeout: 10 * 1500,
```

Comment:
@beganovich Some tests for invoices take longer than 10 seconds, which is the reason for increasing this timeout.